.github/ISSUE_TEMPLATE/1-bug.yml

```diff
@@ -0,0 +1,145 @@
+name: 🐛 Bug report
+description: Describe a problem
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Some general tips:
+
+        - Is this really a problem?
+        - Is this a problem here?
+        - Can this be solved in a different way?
+
+  - type: checkboxes
+    id: initial-checklist
+    attributes:
+      label: Initial checklist
+      options:
+        - label: I read the [support docs](https://mdxjs.com/support)
+          required: true
+        - label: I read the [contributing guide](https://mdxjs.com/contributing)
+          required: true
+        - label: I agree to follow the [code of conduct](https://github.com/mdx-js/.github/blob/HEAD/code-of-conduct.md)
+          required: true
+        - label: I searched [issues](https://github.com/search?q=user%3Amdx-js&type=Issues) and couldn’t find anything (or linked relevant results below)
+          required: true
+
+  - type: input
+    id: affected-versions
+    attributes:
+      label: Affected packages and versions
+      description: Please test using the latest version of the relevant packages to make sure your issue has not already been fixed.
+    validations:
+      required: true
+
+  - type: input
+    id: runnable-example
+    attributes:
+      label: Link to runnable example
+      description: |
+        Link to repository or sandbox with runnable example of the issue.
+
+        Some CodeSandbox starters:
+        - [MDX Evaluate with Create React App](https://codesandbox.io/s/mdx-evaluate-create-react-app-kg630)
+        - [MDX Loader with Create React App](https://codesandbox.io/s/mdx-loader-create-react-app-9e39f)
```

Comment:
MDX Loader v2 and CRA 4 and 5 don't seem to be playing well together.
I'd be interested in ideas/suggestions on how to update the sandbox to resolve the issue (it should in theory work).
Otherwise it can be removed for now.

Reply:
would it be good to maintain starters in git? Such as here?

Reply:
The problem is that these examples were essentially pulling in the whole ecosystem, with all these different server frameworks. And, in our main repo, all build tools are also pulled in for integrations, and client frameworks.
So it made the repo incredibly slow.

If they’re reproduction starters, then I think this repo here makes sense.

Perhaps starters are enough and codesandboxes are enough?

Reply:
Sure, moves these in repo in 0de46eb

The links won't go live until this is merged
but they can be previewed with:
https://codesandbox.io/s/github/ChristianMurphy/.github-mdx/tree/issue-forms/sandbox-templates/mdx-evaluate-create-react-app
and
https://codesandbox.io/s/github/ChristianMurphy/.github-mdx/tree/issue-forms/sandbox-templates/mdx-loader-create-react-app